src/common/transformations/src/ngraph_ops/augru_sequence.cpp

```diff
@@ -0,0 +1,183 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_ops/augru_sequence.hpp"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "itt.hpp"
+
+using namespace std;
+using namespace ngraph;
```

Comment:
Please use new namespace for new operations

Reply:
done